core/flyout.js

```diff
@@ -799,13 +980,23 @@ Blockly.Flyout.prototype.getClientRect = function() {
   // area are still deleted.  Must be larger than the largest screen size,
   // but be smaller than half Number.MAX_SAFE_INTEGER (not available on IE).
   var BIG_NUM = 1000000000;
-  if (this.RTL) {
-    var width = flyoutRect.left + flyoutRect.width + BIG_NUM;
-    return new goog.math.Rect(flyoutRect.left, -BIG_NUM, width, BIG_NUM * 2);
+  var x = flyoutRect.left;
+  var y = flyoutRect.top;
+  var width = flyoutRect.width;
+  var height = flyoutRect.height;
+
+  if (this.toolboxPosition_ == Blockly.TOOLBOX_AT_TOP) {
+    return new goog.math.Rect(-BIG_NUM, y - BIG_NUM, BIG_NUM * 2,
+        BIG_NUM + height);
+  } else if (this.toolboxPosition_ == Blockly.TOOLBOX_AT_BOTTOM) {
+    return new goog.math.Rect(-BIG_NUM, y, BIG_NUM * 2,
+        BIG_NUM + height);
```

Comment:
Just want to double check that this one is right after you fixed the toolbox at bottom overlapping bug.  I think yes since the y coordinate looks right to me, but checking it works in your mulitplayground seems worth doing.